pytest_plugins/rerun_rp/rerun_rp.py

```diff
@@ -64,7 +64,7 @@ def pytest_collection_modifyitems(items, config):
     """
     Collects tests based on pytest option to select tests marked as failed on Report Portal
     """
-    fail_args = config.getvalue('only_failed')
+    fail_args = config.getoption('only_failed', False)
```

Comment:
Is this really required to set False. Isnt it auto understood ?

Since I have tested this plugin with and without  option, it works great for me without setting it to False.

Reply:
Being consistent and explicit with the the pytest API here. So that its not dependent on the default of the option definition.  `config.getoption` will raise ValueError if no default is passed or specified by the option definition.

https://docs.pytest.org/en/stable/_modules/_pytest/config.html#Config.getoption